.github/CODEOWNERS

```diff
@@ -1,2 +1,4 @@
-# Team responsable for Fleet Server
+# Team responsible for Fleet Server
 * @elastic/elastic-agent-control-plane
+
+/deploy/kubernetes @elastic/obs-cloudnative-monitoring
```

Comment:
@elastic/elastic-agent-control-plane I am getting error:
```
Unknown owner on line 4: make sure the team @elastic/obs-cloudnative-monitoring exists and has write access to the repository 
```
who could help me to get write access to this repo?

Reply:
@andresrc this is the one talking about. Is there any docs mentioning the process to change this?

Reply:
yes, it works, now getting: `This CODEOWNERS file is valid. `